src/main/java/org/jabref/gui/fieldeditors/EditorTextArea.java

```diff
@@ -26,7 +28,20 @@ public class EditorTextArea extends TextArea implements Initializable, ContextMe
     };
 
     public EditorTextArea() {
+        // Call the constructor with an empty string.
```

Comment:
Remove comment - the code says it.

You can add a WHY to the comment. Why the empty string is passed. (If you know)